src/Generator/Driver.cs

```diff
@@ -316,14 +316,25 @@ public void SaveCode(IEnumerable<GeneratorOutput> outputs)
                     var fileRelativePath = $"{fileBase}.{template.FileExtension}";
 
                     var file = Path.Combine(outputPath, fileRelativePath);
-                    File.WriteAllText(file, template.Generate());
+                    WriteGeneratedCodeToFile(file, template.Generate());
                     output.TranslationUnit.Module.CodeFiles.Add(file);
 
                     Diagnostics.Message("Generated '{0}'", fileRelativePath);
                 }
             }
         }
 
+        private void WriteGeneratedCodeToFile(string file, string generatedCode)
+        {
+            var fi = new FileInfo(file);
+
+            // We only compare contents if the generated code length is the same as the length of the existing file. If their lengths are different then they are
```

Comment:
Please excuse me as I'm a little distracted today. Allow me to ask you for one final change: delete this comment as your code is self explanatory.

Reply:
No problem. Done